src/logic/render/LineRenderEngine.ts

```diff
@@ -58,6 +58,8 @@ export class LineRenderEngine extends BaseRenderEngine {
                 store.dispatch(updateActiveLabelId(labelLineUnderMouse.id));
             } else if (!this.isInProgress() && isMouseOverImage) {
                 this.startNewLabelCreation(data)
+            } else if (!this.isInProgress()) {
+
             } else {
```

Comment:
Thank you for pointing out this bug. I would suggest another solution to this problem. We will correct this line.
```suggestion
            } else if (isMouseOverImage) {
```
I guess it should work?

Reply:
This will lose a feature（The ending point can be outside the picture and in canvas），you can try it

Reply:
another way
```diff
-            } else if (!this.isInProgress() && isMouseOverImage) {
-                this.startNewLabelCreation(data)
+            } else if (!this.isInProgress()) {
+                isMouseOverImage && this.startNewLabelCreation(data)
```

Reply:
I'll take a look at it ever the weekend and provide some suggestion :) (or just agree with you)

Reply:
Ok I took a look, and I would propose changing line 61 into: 
```suggestion
            } else if (this.isInProgress()) {
```

Reply:
#167